### ISSUE TITLE
Enormous delay after building 

### ISSUE THREAD

**Issue by skypanther** [sharpred/grunt-appc-cli#8]
I'm using the grunt script at https://gist.github.com/skypanther/b61a9e0522c56a04258b104966d9d1e1 to automate builds. After building for iOS, my system sits at the following lines for a good 20-30 sec before moving on to build for Android; after the APK is built, again, it sits there for 20-30 sec before ending and dropping me back to the prompt.

```
>> [INFO]  Package location: /Users/skypanther/myproject/dist/DEMO.ipa
>> [INFO]  Project built successfully in 1m 19s 414ms
```

I've scanned the code of this project and don't see that your code is doing anything in that time. And my grunt script isn't. What is it doing? I never had this issue when using just the `ti` commands.

**Comment by sharpred** [comment]
Hi Tim

Thanks for using my module and for bringing this to my attention.  Finger of blame looks to be pointing at node child_process spawn.  It looks like the 'close' event is not being emitted or is emitting way too late.

I could refactor this to use child_process exec

As a quick and dirty fix I amended the listener on the data emitter to the following, which works

`task.stdout.on('data', function(data) {
            if(data.indexOf("Project built successfully") !== -1) {
                done(true);
            }
            grunt.log.ok(data);
        });`

YMMV, but am happy to commit this change.  I don't have time to do the exec change just now.  Let me know what you think.

**Comment by sharpred** [comment]
have published this change as 1.0.6

**Comment by skypanther** [comment]
Thanks! The quick-n-dirty fix worked just fine. No more delay. Thanks for publishing this module!
